packages/db-objects-redis/lib/objects/objectsInRedisClient.js

```diff
@@ -393,6 +395,16 @@ class ObjectsInRedisClient {
                 });
             }
 
+            try {
```

Comment:
Ich glaube das muss in eine red "ready" events rein ... weil die DB ist noch nicht initialisiert an der code stelle oder?! Lso ja kann gehen wegen queued befehlen ...

Reply:
it is but I've now put it 5 lines below under the !this.client check